wallet-new/src/Cardano/Wallet/API/V1/Types.hs

```diff
@@ -1489,13 +1518,15 @@ instance ToSchema NodeInfo where
       & ("syncProgress"          --^ "Syncing progression, in percentage.")
       & ("blockchainHeight"      --^ "If known, the current blockchain height, in number of blocks.")
       & ("localBlockchainHeight" --^ "Local blockchain height, in number of blocks.")
-      & ("localTimeDifference"   --^ "Local time difference, in number of blocks.")
+      & ("localTimeInformation"  --^ "Information about the clock on this node.")
     )
 
 instance Arbitrary NodeInfo where
+    -- TODO(matt.parsons): The Swagger spec doesn't understand optional/maybe values for
+    -- some reason, so we create Just values instead.
+    --
+    -- This is fixed in this issue: https://github.com/GetShopTV/swagger2/issues/142
     arbitrary = NodeInfo <$> arbitrary
```

Comment:
The issue seems to only concerns unary records, right ?

Reply:
Yes. We could add another field and work around it, but I'd rather see about bumping the swagger dependency.

Reply:
But isn't the comment & the `fmap Just` redundant with the instance (and comment) of `TimeInfo` though?

Reply:
oh, that may be the case! i am investigating. good call :smile: